app/commands/contact_publisher.rb

```diff
@@ -0,0 +1,12 @@
+module Commands
+  class ContactPublisher
+    def initialize(contact)
+      @contact = contact.reload
```

Comment:
This feels a little surprising that a model is mutated in an initializer, I think this should be the responsibility of the caller.